app/helpers/admin_helper.rb

```diff
@@ -7,6 +7,21 @@ def sort_link(column, title = nil)
     link_to "#{title} <span class='#{icon}'></span>".html_safe, {column: column, direction: direction}.merge(params.permit(:type, :search))
   end
 
+  def set_mentions(note)
+    # Regex to find any @words
+    note.scan(/\@(\w*)/).each do |mention|
```

Comment:
Do you think there's a way to do this without a regex?

Reply:
The only other way I could think of doing this without regex was splitting on spaces and checking through each word if it started with an `@`. 

Do you think that would be more readable?

Reply:
Updated because I wanted to add link functionality 🎉 
https://github.com/brave-intl/publishers/pull/1896/commits/6b1b6e71364d5b8d1953b571b364ee13e9b44f99